docs/narr/testing.rst

```diff
@@ -50,7 +50,7 @@ The suggested mechanism for unit and integration testing of a :app:`Pyramid`
 application is the Python :mod:`unittest` module.  Although this module is
 named :mod:`unittest`, it is actually capable of driving both unit and
 integration tests.  A good :mod:`unittest` tutorial is available within `Dive
-Into Python <http://www.diveintopython.net/unit_testing/index.html>`_ by Mark
+Into Python <https://www.diveinto.org/python3/unit-testing.html>`_ by Mark
```

Comment:
This is now "Dive into Python 3". Would you please make this additional correction? Thank you!

Reply:
will do.